src/ex3/index.js

```diff
@@ -0,0 +1,112 @@
+import { Command } from 'commander';
+import Inquirer from 'inquirer';
+import * as ItemManager from './ItemManager.js';
+
+const program = new Command();
+
+function addTasksCli(args,options) {
```

Comment:
it's a convention to have `options` argument as the last argument (as you did), but **optional**, so you better give it a default value in case it's not given:
```suggestion
function addTasksCli(args, options = {}) {
```

Reply:
thanks, done!